Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -25332,4 +25359,209 @@
         <EditorCategories value="ObjectType:Structure"/>
         <HotkeyCategory value="Unit/Category/AP_ZergUnits"/>
     </CUnit>
+    <CUnit id="AP_MercZergling" parent="AP_Zergling">
+        <Name value="Unit/Name/AP_MercZergling"/>
+        <Description value="Button/Tooltip/AP_MercZergling"/>
+        <LifeStart value="120"/>
+        <LifeMax value="120"/>
+        <AbilArray index="4" Link="AP_BurrowMercZerglingDown"/>
+        <AbilArray index="5" Link="AP_HotSRaptorCharge"/>
+        <AbilArray index="6" removed="1"/>  <!--move raptor abil up and remove baneling morph-->
+        <BehaviorArray index="0" Link="AP_HotSRaptorJump"/>
+        <BehaviorArray index="1" Link="AP_MalignantCreepFix"/>
+        <BehaviorArray index="2" removed="1"/>  <!--move raptor behaivor up and remove respawn ability-->

```

Comment:
You can do `<AbilArray index="4" Link=""/>` in order to remove an element from the middle. it's easier to maintain

Reply:
Good to know, that's easier.